src/pages/login.rs

```diff
@@ -155,9 +155,9 @@ pub fn Login(cx: Scope) -> Element {
                         is_loading_loggedin.set(LoggedInStatus::Start);
                         return;
                     };
-                }else {
+                } else {
                     let result = auth.set_server(homeserver.get()).await;
-                    log::error!("Failed to set server: {result:?}")
+                    log::info!("Failed to set server: {result:?}")
```

Comment:
I still think it doesn't make much sense to log Ok and Err equally, specially if the message implies failure, what does it mean to log every time it succeeds, "info: Failed to set server:  Ok(...)" ? the Ok might not need a log and the if the error is not very important it could be a warn! ? 
```rust
if let Err(e) = auth.set_server(homeserver.get()).await {
    log::warn!("Failed to set server: {e}");
} 
```